20151014/idl/height.pro

```diff
@@ -0,0 +1,40 @@
+; find the height in solar radii corresonding to frequency
+
+pro height
+
+  ; read frequency data
+  readcol, 'UpperBand.dat', f = 'A', time_upper, fit_upper, sec_upper   
+  readcol, 'LowerBand.dat', f = 'A', time_lower, fit_lower, sec_lower
+
+  fit_upper = (fit_upper - 4.58) * 1.E6	;converting MHz to Hz
+  fit_lower = (fit_lower - 3.89) * 1.E6	
+  
+  ; Density model
+  restore,'density_zucca_et_al_01062015.sav'
+
+  ; convert frequency (Hz) to density (cm^-3): f = 18000 sqrt(ne)
+  density_upper = ( fit_upper/ 18000 ) ^ 2
+  density_lower = ( fit_lower/ 18000 ) ^ 2
```

Comment:
Even though `fit_upper` is a float, if it was integer the result would also be an integer.
Suggesting 
```idl
density_upper = ( fit_upper/ 18000. ) ^ 2
```